### ISSUE TITLE
date format of Other record in Systems table

### ISSUE THREAD

**Issue by kdahlquist** [lmu-bioinformatics/xmlpipedb#9]
So the date format issue has moved records.  In the full gdb that completed last night, the record with the date and time instead of just the date is not GeneOntology, but Other.  So I'm thinking it might have something to do with the last record updated in that table?

**Comment by dondi** [comment]
This issue has been fixed on my local branch; I figured a release can wait pending confirmation that the TallyEngine discrepancies are OK, plus the possibility that the new link-outs can be included. If you prefer to release a build now with just this fix, let me know and I can do that too.

(I'll leave a review requested tag to mark that this issue is potentially addressed)

**Comment by kdahlquist** [comment]
I updated the link outs in issue #10 so if you could make the changes, that would be great.

**Comment by dondi** [comment]
This fix has been released as **build-5**. Will wait on a little more testing and usage before closing.

**Comment by dondi** [comment]
Merged into `master` via pull request #17.
